test/reuse/helper/errorHandler/logException.spec.js

```diff
@@ -4,23 +4,23 @@ const ErrorHandler = require("../../../../lib/reuse/helper/errorHandler").defaul
 
 describe("errorHandler - logException - error message without stacktrace", async function () {
   it("Execution & Verification", async function () {
-    const errorHandler = new ErrorHandler(false);
+    const errorHandler = new ErrorHandler();
     await expect(async function display() {
-      errorHandler.logException(new Error("Divide by zero error."));
+      errorHandler.logException(new Error("Divide by zero error."), undefined, false);
     }).rejects.toThrow("Function 'display' failed with : Divide by zero error.");
   });
 });
 
-describe("errorHandler - logException - error message with stacktrace by passing constructor value true", async function () {
-  const errorHandler = new ErrorHandler(true);
+describe("errorHandler - logException - error message with stacktrace by passing stacktrace parameter value true", async function () {
+  const errorHandler = new ErrorHandler();
   it("Execution & Verification", async function () {
     await expect(async function display() {
-      errorHandler.logException(new Error("File not found exception"));
+      errorHandler.logException(new Error("File not found exception"), undefined, true);
     }).rejects.toThrow(/Function 'display' failed with : File not found exception/);
```

Comment:
do you also assert a part of the stack trace?

Reply:
done

Reply:
done